plugins/grpc-transport/src/main/java/com/github/bsideup/liiklus/transport/grpc/config/GRPCConfiguration.java

```diff
@@ -20,8 +21,11 @@
 import java.util.concurrent.TimeUnit;
 
 @AutoService(ApplicationContextInitializer.class)
+@Order(GRPCConfiguration.GRPC_CONFIGURATION_ORDER)
```

Comment:
since you read `GRPCTransportConfigurer`s only in `registerBean`'s factory lambda, there shouldn't be a need to have a strict order. All configurations will register the beans before the resolve will start, and, by the time you call `getBeansOfType(GRPCTransportConfigurer.class)`, all configurers should be available

Reply:
oh, didn't know that, thx

Reply:
worth testing tho ;)